### PR TITLE
Added Japanese language

### PR DESCRIPTION
**Pull request by ringo360**
Japanese translation added.
> This translation includes additional explanations for parts of the default (en_US) explanation that were difficult to understand. (For example, the ProgressBarLength and Settings sections)

### CODE REVIEW

File: ja_JP/config.yml [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+      Cramping: 6
+      Color:
+        High: "&a"
+        Medium: "&e"
+        Low: "&c"
+  Villager:
+    Enabled: true
+
+#InteractionVisualizerディスプレイのために、これらのモジュールを有効にします。
+#ItemStand は、アーマースタンドとアイテムフレームを使用するすべてのディスプレイを表します。
```
Comment (takejohn) [inline ja_JP/config.yml:220]:
```suggestion
#ItemStand は、防具立てと額縁を使用するすべてのディスプレイを表します。
```
*armorstands*, *itemframes* は日本語版 Minecraft においてそれぞれ「防具立て」、「額縁」と呼ばれることが一般的です。

File: ja_JP/music.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  BASS_DRUM: バスドラム
+  BASS_GUITAR: バスギター
+  BELL: ベル
+  BIT: ビット(電子音)
+  CHIME: チャイム
+  COW_BELL: カウベル
+  DIDGERIDOO: ディジュリドゥ
+  FLUTE: フルート
+  GUITAR: ギター
+  IRON_XYLOPHONE: 鉄筋
```
Comment (takejohn) [inline ja_JP/music.yml:13]:
```suggestion
  IRON_XYLOPHONE: 鉄琴
```
typo: `鉄筋` -> `鉄琴`

**Review by takejohn — CHANGES_REQUESTED** [review]
Nice translation!
However, I saw some typos. Please check them.

File: ja_JP/config.yml (hunk 1 above)
Comment (ringo360) [reply ja_JP/config.yml:220]: Thanks <3

**Comment by ringo360** [comment]
Please merge…

**Pull request merged**